dataland-frontend/src/utils/SessionTimeoutUtils.ts

```diff
@@ -17,21 +17,19 @@ const minRequiredRemainingValidityTimeOfRefreshTokenDuringCheck = TIME_DISTANCE_
  * @param forceStoringValues forces storing the refresh token and its expiry timestamp in the shared storage, even if
  * the updateToken() has not done an update itself because of the minValidity value
  */
-export function updateTokenAndItsExpiryTimestampAndStoreBoth(keycloak: Keycloak, forceStoringValues = false): void {
-  keycloak
-    .updateToken(60)
-    .then((hasTokenBeenUpdated) => {
-      if (hasTokenBeenUpdated || forceStoringValues) {
-        const refreshTokenExpiryTime = keycloak.refreshTokenParsed?.exp;
-        if (refreshTokenExpiryTime) {
-          useSharedSessionStateStore().refreshToken = keycloak.refreshToken;
-          useSharedSessionStateStore().refreshTokenExpiryTimestampInMs = refreshTokenExpiryTime * 1000;
-        }
-      }
-    })
-    .catch(() => {
-      console.log(`Could not refresh token`);
-    });
+export async function updateTokenAndItsExpiryTimestampAndStoreBoth(
+  keycloak: Keycloak,
+  forceStoringValues = false,
+): Promise<void> {
+  const hasTokenBeenUpdated = await keycloak.updateToken(60);
+
+  if (hasTokenBeenUpdated || forceStoringValues) {
+    const refreshTokenExpiryTime = keycloak.refreshTokenParsed?.exp;
+    if (refreshTokenExpiryTime) {
+      useSharedSessionStateStore().refreshToken = keycloak.refreshToken;
+      useSharedSessionStateStore().refreshTokenExpiryTimestampInMs = refreshTokenExpiryTime * 1000;
+    }
+  }
```

Comment:
Reminder for myself to ask Marc:

This function seems to do the same thing as before. The only differences seem to be:
a) It does not catch errors anymore
b) It is using a different syntax (async instead of .then-chaining)

So why the adjustments?